launcher/modplatform/flame/FlameInstanceCreationTask.cpp

```diff
@@ -404,6 +411,39 @@ void FlameCreationTask::idResolverSucceeded(QEventLoop& loop)
     }
 }
 
+/// @brief copy the matched blocked mods to the instance staging area
+/// @param blocked_mods list of the blocked mods and their matched paths
+void FlameCreationTask::copyBlockedMods(QList<BlockedMod> const& blocked_mods) {
+
+    setStatus(tr("Copying Blocked Mods..."));
+    setAbortable(false);
+    int i = 0;
+    int total = blocked_mods.length();
+    setProgress(i, total);
+    for (auto &mod : blocked_mods) {
```

Comment:
you might want to run clang-format on your modifications, there are some inconsistencies in the formatting

Reply:
you can also add the const there ^^
```suggestion
    for (auto const& mod : blocked_mods) {
```